pallets/runtime/src/test/storage.rs

```diff
@@ -310,6 +301,7 @@ pub type Randomness = pallet_randomness_collective_flip::Module<TestStorage>;
 pub type Timestamp = pallet_timestamp::Module<TestStorage>;
 pub type Contracts = pallet_contracts::Module<TestStorage>;
 pub type Bridge = bridge::Module<TestStorage>;
+pub type CDDServieProvider = group::Module<TestStorage, group::Instance2>;
```

Comment:
Possibly you meant `CddServiceProvider` ;-)

Please convert this identifier to Pascal-case, that is `Cdd` rather than `CDD`.